.github/workflows/appimage.yaml

```diff
@@ -0,0 +1,62 @@
+name: Build AppImage
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
```

Comment:
@Abs62 - This "on" section tells GitHub when to run this build.  Right now I have it set to run "when a maintainer asks for it" ("workflow_dispatch") and when "master" branch is pushed. You can also set it up to run on pull requests, new releases, etc.  See the full set of triggering option here: https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows